contents/blog/collaborating-on-posthog.md

```diff
@@ -0,0 +1,86 @@
+---
+date: 2022-03-24
+title: Collaborating on PostHog
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["paolodamico"]
+featuredImage: ../images/blog/generic-release-notes.png
+featuredImageType: full
+categories: ["Product updates"]
+---
+
+PostHog is better with your team. We quickly realized that as more users join an organization, the incremental value derived by each user increases. Consider for instance when you create a dashboard. You only create it once, but then with each user that joins your organization, the dashboard becomes more valuable. We found clear signals of this in our data: organizations with multiple teammates retain on average 30%+ better. We also recognized that conservatively we could increase insight discovery (our [main Product metric](https://posthog.com/handbook/product/metrics)) by more than 20% by improving collaboration workflows [1]. For instance, we discovered that 50%+ of invited teammates never discover a single insight. If we could get these invited teammates properly onboarded, the overall value derived from PostHog would increase significantly.
+
+
+Armed with the above we set out to make collaboration in PostHog as easily as possible, particularly gearing our efforts to making sure that larger teams are successful, with growing pains that tend to be negligible in small teams. We'll definitely continue improving on this, but we want to show you what we have today.
```

Comment:
```suggestion
PostHog is better with teammates. You only have to create a dashboard once, but it becomes more valuable every time you invite someone from your organization to view it. And this isn't just some vague truism: it's a measurable fact.

Our data shows organizations with multiple teammates retain 30%+ better than those with a single user. Better collaboration also powers our [main Product metric](https://github.com/handbook/product/metrics): insight discoveries. Our conservative estimates suggest we could increase discoveries by 20% by improving collaboration workflows [1], so that's what we've done.

Over the past few months, we've worked hard to make collaboration in PostHog as easy as possible, particularly for larger teams. That work will never end, but we want to show you what we have today.
```

Have rejigged the opening paras a little to tighten things up.